Lib/test/test_embed.py

```diff
@@ -1318,10 +1318,11 @@ def test_init_pybuilddir(self):
         with self.tmpdir_with_python() as tmpdir:
             # pybuilddir.txt is a sub-directory relative to the current
             # directory (tmpdir)
+            vpath = sysconfig.get_config_var("VPATH") or ''
             subdir = 'libdir'
             libdir = os.path.join(tmpdir, subdir)
             # The stdlib dir is dirname(executable) + VPATH + 'Lib'
-            stdlibdir = os.path.join(tmpdir, 'Lib')
+            stdlibdir = os.path.normpath(os.path.join(tmpdir, vpath, 'Lib'))
```

Comment:
Currently, this actually does nothing as `VPATH` is implemented only on Windows. `sysconfig.py` and `sysmodule.c` also need to be changed if necessary. Otherwise, I think keeping the status quo is less confusing.

Reply:
Removed windows.

Reply:
Correct, VPATH is calculated by the configure script on Linux, and hard coded on Windows. But it only has any relevance for local builds - once a build has been properly installed, it's irrelevant.

I'm not sure exactly how it works in out-of-tree builds, but possibly it's supposed to map back from the build directory to the repository?